config.toml

```diff
@@ -28,7 +28,7 @@ relativeURLs = false
     url = "https://delphi.cmu.edu/blog"
     weight = 4
 [[menu.main]]
-    name = "GitHub"
+    name = "Github"
```

Comment:
GitHub brands itself GitHub; I think we should actually change the other site to match this, rather than changing the blog to be wrong